packages/playground/src/App.js

```diff
@@ -290,6 +295,12 @@ export default class extends React.Component {
           >
             Start conversation with local MockBot
           </button>
+          <button
+            onClick={ this.handleDisconnectClick }
+            type="button"
```

Comment:
nit: this is redundant.

Reply:
Kind of, until we fix our reconnection story.

This is currently for Corina to test her connectivity status UI.